### DISCUSSION TITLE
[autorefresh] docker + freshRSS

### DISCUSSION THREAD

**Discussion by toyo2333** [FreshRSS/FreshRSS#5617] [Q&A]
i set CRON_MIN: '*/20'  in the docker-compose,yml 

but i find “Archiving” -》Do not automatically refresh more often than is “1 hour” by default

so my question is Which setting really works in my case?

**Comment by Frenzie** [comment]
Both really work. ;-) But assuming you mean regarding the maximum frequency of feed updates, the latter.

**Comment by toyo2333** [comment]
> Both really work. ;-) But assuming you mean regarding the maximum frequency of feed updates, the latter.

it is confused，hard to understand...

**Comment by Frenzie** [comment]
A cron job (scheduled job) starts an action, as in some script or application. In this case that script checks for updates.

If a particular feed was last downloaded less than `Do not automatically refresh more often than` the script skips it and moves on to the next one because there's nothing to do there yet.
